### ISSUE TITLE
When fine-tuning omni-llm-7B, the loss is abnormal.

### ISSUE THREAD

**Issue by Liuwangtao** [facebookresearch/omnilingual-asr#43]
When I fine-tune the omni-llm-7B model with multilingual ASR data, the loss is abnormal: the loss for the first 200 steps is around 13. However, when I fine-tune the omni-llm-3B model with the same data, the loss for the first 200 steps is around 0.5. Below is my yaml：

model:
  name: "omniASR_LLM_7B"
  dtype: "torch.bfloat16"

dataset:
  name: "six_msr"
  train_split: "train"
  valid_split: "dev"
  storage_mode: "MIXTURE_PARQUET"
  task_mode: "ASR"
  mixture_parquet_storage_config:
    dataset_summary_path: "/mnt/ufs/asr_data/user/xxx/omnilingual-asr/stats.tsv"
    beta_corpus: 0.5
    beta_language: 0.5
    fragment_loading:
      cache: True
  asr_task_config:
     min_audio_len: 4_800
     max_audio_len: 480_000
     batch_size: 8
     num_seqs_multiple_of: 8
     max_num_elements: 480_000
     batch_shuffle_window: 1
     normalize_audio: true
     example_shuffle_window: 1
     no_padding: true

tokenizer:
  name: "omniASR_tokenizer"

optimizer:
  config:
    lr: 5e-05

trainer:
  freeze_encoder_for_n_steps: 0
  mixed_precision:
    dtype: "torch.bfloat16"
  grad_accumulation:
    num_batches: 1

regime:
  num_steps: 2_000_000
  num_data_epochs: null
  validate_after_n_steps: 0
  validate_every_n_steps: 2_000_000
  checkpoint_every_n_steps: 20_000
  publish_metrics_every_n_steps: 200
common:
  sweep_format: omniasr_llm_7B
